src/Sentry/SessionMode.cs

```diff
@@ -0,0 +1,18 @@
+namespace Sentry
+{
+    /// <summary>
+    /// Determines the Session manager type.
+    /// </summary>
+    public enum SessionMode
+    {
+        /// <summary>
+        /// The Client.
+        /// </summary>
+        Client,
+
+        /// <summary>
+        /// The Server.
```

Comment:
```suggestion
        /// Server mode. Also known as Request Mode.
```

Add `<summary>` describing a bit how this works? Also that's default on Sentry.AspNetCore and Sentry.AspNet?

Similar on the Client mode, how it works. Each call will send a session update immediately to Sentry etc